gammapy/visualization/tests/test_utils.py

```diff
@@ -43,14 +44,30 @@ def test_plot_spectrum_datasets_off_regions():
     assert ax.lines[0].get_color() in ["green", "C0"]
 
 
-def test_map_panel_plotter():
+def test_map_panel_plotter(caplog):
+    caplog.set_level(logging.INFO)
     t = np.linspace(0.0, 6.1, 10)
     x = np.cos(t)
     y = np.sin(t)
 
     ax = plt.subplot(111)
     with mpl_plot_check():
         plot_contour_line(ax, x, y)
+        assert "INFO" in [_.levelname for _ in caplog.records]
```

Comment:
Note that the test will fail once we upgrade to the latest iminuit version. So this does not allow transparent usage of versions.

Reply:
The test on the logging is removed. I am wondering what there is an issue with the latest version of iminuit... Can you provide more information? Thanks...

Reply:
In iminuit 2.13 contours are returned already closed contrary to <=2.12, so that the behaviour changes between the two versions.